cczoo/rats-tls/README.md

```diff
@@ -0,0 +1,176 @@
+# Building
+
+## Build Requirements
+
+- git
+- make
+- autoconf
+- libtool
+- gcc
+- g++ (ubuntu 18.04)
+- SGX driver, Intel SGX SDK & PSW: Please refer to this [guide](https://download.01.org/intel-sgx/sgx-linux/2.14/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf) to install.
+- [SGX DCAP](https://github.com/intel/SGXDataCenterAttestationPrimitives): please download and install the packages from this [page](https://download.01.org/intel-sgx/sgx-dcap/#version#linux/distro).
+  - ubuntu 18.04: `libsgx-dcap-quote-verify-dev`, `libsgx-dcap-ql-dev`, `libsgx-uae-service`
+- For TDX, please see the README in TDX MVP Stack. You need to download the packages and following TDX_E2E_attestation_software_stack_Installation_README-dcap-2021XXXX.txt to do step 2 & step 3 to setup build and dependence libraries.
+
+## Build and Install
+
+Please follow the command to build RATS TLS from the latested source code on your system.
+
+1. Download the latest source code of RATS TLS
+
+```shell
+mkdir -p "$WORKSPACE"
+cd "$WORKSPACE"
+git clone https://github.com/inclavare-containers/rats-tls
```

Comment:
Should this github address be changed to cczoo?

Reply:
In case rats-tls here is exactly the same as inclavare-containers/rats-tls, it looks better to include rats-tls as a git submodule